airbyte-integrations/connectors/destination-bigquery/build.gradle

```diff
@@ -1,16 +1,13 @@
 plugins {
-    id 'application'
     id 'airbyte-java-connector'
 }
 
 airbyteJavaConnector {
-    cdkVersionRequired = '0.15.1'
-    features = ['db-destinations', 's3-destinations', 'typing-deduping']
-    useLocalCdk = false
+    cdkVersionRequired = '0.17.0'
+    features = ['db-destinations', 's3-destinations', 'typing-deduping', 'datastore-bigquery']
+    useLocalCdk = true
```

Comment:
destination-bigquery needs to be in this PR because it used to depend on destination-gcs, but no longer does.